### PR TITLE
Update storj-sim documentation

### PR DESCRIPTION
**Pull request by egonelbre**

### CODE REVIEW

File: Test-network.md [hunk 1]
```diff
@@ -1,12 +1,11 @@
...
-The Storj test network (storj-sdk) enables you to run all the components of the Storj platform (Satellite, Uplink client, and storage nodes) and test them on your local machine.
+The `storj-sim` tool enables you to run all the components of the Storj network (Satellite, Storage Nodes, Console and Gateway) and test them on your local machine.
 
 ![network components](assets/components.png)
 
-In every day Storj usage, the Satellite, Storage node, and Uplink are run
-separately, but for the test network, all three components are set up and run
-in the same process.
+In every day Storj usage, the Satellite, Storage Node, and Uplink are run
+on separate servers and computers, but for the test network, all of the components are running locally.
```
Comment (keleffew) [inline Test-network.md]: change to "but for the purposes of the test network, all of the three components are run locally."

File: Test-network.md [hunk 2]
```diff
@@ -43,42 +44,43 @@ The next step is to run it!
...
 
 ### Wiping the Testnet
 
-`storj-sdk network destroy` can be used to wipe the network easily. e.g. it can be convenient to run the command in a single line, like so:
+`storj-sim network destroy` can be used to wipe the network easily.
+
+While developing it's often nice to be able to delete the network and set it up from scratch.
 
-`storj-sdk network destroy && storj-sdk network setup && storj-sdk network test bash my-test-script.sh`
+It can be very convenient to run the command in a single line, like so:
```
Comment (keleffew) [inline Test-network.md]: Change to "For convenience, you may run the command in a single line, like so:"

File: Test-network.md [hunk 3]
```diff
@@ -16,25 +15,27 @@ installed run:
...
 
 You now have a configured Storj test network with default configuration options.
 
-You might also want to take a look at the config by navigating to the root directory `--config-dir` where all the configs are specified.
+You might also want to take a look at the config by navigating to the root 
+directory `--config-dir` where all the configs are specified.
+You can tweak the configuration settings there as needed.
 
-You can use vim to tweak the default configuration settings. You can also see what is being overwritten on the command-line level with `storj-sdk -x network run`.
+To see what is happening under the hood you can use `-x` or `--print-commands` to see how the processes are exactly started.
```
Comment (keleffew) [inline Test-network.md]: Change to:  For insight into what is happening under the hood you can use `-x` or `--print-commands` to see how the processes are started

File: Test-network.md (hunk 1 above)
Comment (egonelbre) [reply Test-network.md]: There are more components and will probably change in the future.

**Pull request merged**